src/main/kotlin/it/pagopa/wallet/client/WalletQueueClient.kt

```diff
@@ -23,8 +25,12 @@ class WalletQueueClient(
         tracingInfo: QueueTracingInfo
     ): Mono<Response<SendMessageResult>> {
         val queueEvent = QueueEvent(event, tracingInfo)
-        return BinaryData.fromObjectAsync(queueEvent, jsonSerializer).flatMap {
-            cdcQueueClient.sendMessageWithResponse(it, delay, ttl)
-        }
+        return BinaryData.fromObjectAsync(queueEvent, jsonSerializer)
+            .flatMap { cdcQueueClient.sendMessageWithResponse(it, delay, ttl) }
+            .retryWhen(
+                Retry.backoff(3, Duration.of(1, ChronoUnit.SECONDS)).doBeforeRetry { signal ->
+                    println("Retrying due to: ${signal.failure().message}")
```

Comment:
use logger here instead of printing message on system out

Reply:
solved in 636aab273107aba783c1877175d35d73b2ce0a38